### ISSUE TITLE
Dead project? - Definitely not!

### ISSUE THREAD

**Issue by D4nte** [chatscope/chat-ui-kit-react#68]
Interesting project but no commit since last year, just wanted to check the maintenance status. Cheers.

**Comment by supersnager** [comment]
@D4nte Thanks for the question.

I must admit that library development has slowed down recently.

However, the project is __not__ dead. 

In addition to the UI library, the project includes a use-chat library, and a server that will be released as open source (hopefully this year). There will also be some other new toys.

Taking care of all this requires time, which is always too little :)
Some elements that go into the project are often tested earlier in my commercial implementations, which also takes a lot of time.
I also hope to eventually get someone permanently engaged to help with the project.

To sum up, you can be calm, if you want to use the library you will not be left without care.

Cheers!

**Comment by D4nte** [comment]
Thanks!
